sdk/textanalytics/azure-ai-textanalytics/samples/sample_analyze_sentiment_with_opinion_mining.py

```diff
@@ -33,7 +33,7 @@
 
     Since these reviews seem so mixed, and since I'm interested in finding exactly what it is about my hotel that should be improved, let's find the complaints users have about individual aspects of this hotel
```

Comment:
idk if you would like to rename aspect here to target. same in other descriptions of the file.
I think either way is fine, but pointing out just in case